src/sidebar/components/Menu.tsx

```diff
@@ -201,21 +201,16 @@ export default function Menu({
         aria-label={title}
         title={title}
       >
-        <span
-          // wrapper is needed to serve as the flex layout for the label and indicator content.
-          className="flex items-center gap-x-1"
```

Comment:
It turns out this wrapper is actually not needed, as the button itself is already a flex container. Only the `gap-x-1` had to be set there.